views/about.ejs

```diff
@@ -3,7 +3,48 @@
 	<%- include('partials/header'); %>
 	<main class="mdl-layout__content">
 		<div class="mdl-typography--text-center clemson-esports">
-			<h1>Clemson Esports</h1>
+			<h1>About Clemson Esports</h1>
 		</div>
+		<div class="mdl-typography--text-center clemson-esports">
+			<p>
```

Comment:
These paragraphs of text in desktop view should be constrained to having a maximum width, like the Wix site. Otherwise, they can become hard to read.

Reply:
Text has been limited to more reasonable width
![image](https://user-images.githubusercontent.com/24688719/162633925-2574ba2c-a7e8-4e98-a79e-a4d3670927c8.png)